src/Scanner.java

```diff
@@ -1492,6 +1492,7 @@ ChannelBuffer serialize(final byte server_version) {
       final ScanRequest req = ScanRequest.newBuilder()
         .setScannerId(scanner_id)
         .setCloseScanner(true)
+        .setNumberOfRows(0)
```

Comment:
this is specifically for closeScannerRequest, which expect 0 row as a response. not hundred percent sure the internal change, but based on the bug fix log of hbaseAsync, the format changed some where between 0.94 to 0.98. It does solved my exception.